cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -589,7 +590,46 @@ def test_attribute_override():
     Bugzilla:
         1651099
     """
-    pass
+    attributes = [
+        ("class", "Request"),
+        ("instance", "TestNotification"),
+        ("message", "digitronik_msg"),
+        ("namespace", "/System"),
+    ]
+    req = "call_instance_with_message"
+    executions = {"simulation": "create", "execution": "digitronik_msg"}
+
+    collection = appliance.collections.button_groups
+    group = collection.create(
+        text="grp_{}".format(fauxfactory.gen_alphanumeric(3)),
+        hover="hover_{}".format(fauxfactory.gen_alphanumeric(3)),
+        type=getattr(collection, "PROVIDER"),
+    )
+    request.addfinalizer(group.delete_if_exists)
+
+    button = group.buttons.create(
```

Comment:
Is there a reason you create these objects in the test rather than in fixtures? Talking about `button` and `group`.

Reply:
Yes, `button` object more specific about the test.
while writing the test. what we are testing; it should be part of test not fixture. Here we, are testing `attribute override` and attributes are passing while creating button please have a glance on `L608`.

Yup, a group we can move to fixture; we have one which I'm using where we can pass type of custom button.

Reply:
That makes sense, thanks for the explanation.